### PR TITLE
Make Blazectron work

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/ComponentsElectron/src/Microsoft.AspNetCore.Components.Electron/electron-js/api/browserWindows.js [hunk 1]
```diff
@@ -156,6 +156,7 @@ module.exports = function (socket) {
     });
     socket.on('createBrowserWindow', function (options, loadUrl) {
         var window = new electron_1.BrowserWindow(options);
+        window.webContents.openDevTools();
```
Comment (danroth27) [inline src/ComponentsElectron/src/Microsoft.AspNetCore.Components.Electron/electron-js/api/browserWindows.js]: Remove?

**Comment by MaherJendoubi** [comment]
FYI https://github.com/ElectronNET/Electron.NET/releases/tag/0.0.11

**Comment by dougbu** [comment]
According to https://dev.azure.com/dnceng/public/_build/results?buildId=179167 (the validation build for this PR) and https://dev.azure.com/dnceng/public/_build/results?buildId=179476 (the subsequent public CI build), this PR broke compilation of `ElectronRenderer`. I suspect we need a newer SDK so that it includes Microsoft.AspNetCore.Components.Server.

In other words, 'master' in this repo is busted. Please do the needful @pranavkm

**Comment by SteveSandersonMS** [comment]
Thanks for updating this, @pranavkm!

**Pull request merged**